src/frontend/src/flows/dappsExplorer/teaser.ts

```diff
@@ -56,7 +56,6 @@ const marqueeList = (): TemplateResult => {
             src=${logo}
             alt="${name}"
             class="c-marquee__image"
-            loading="lazy"
```

Comment:
can you add a comment about it?

pretty sure that if I land on it in the future I would ask my self why it is not lazy loaded since it isn't my default approach.